setup.py

```diff
@@ -28,7 +28,7 @@ def _get_version():
     ),
     data_files=[
         ('/etc/ubuntu-advantage', ['uaclient.conf']),
-        ('/usr/share/keyrings', glob.glob('keyrings/*')),
+        ('/usr/share/keyrings', glob.glob('debian/keyrings/*')),
```

Comment:
any particular reason for this change? If we ever make this package a non-native package, we will have to revert this, because setup.py would be referencing something that is not part of the tarball anymore.

Reply:
My thought process was since the keyring is getting installed at install time and not something the python code, I would put it in the debian dir.

I can pull it out if that makes more sense?

Reply:
It's not incorrect, just not needed. We can leave it as is.